assets/docs/configuration/transformations/builtin/jq-full-example.hcl

```diff
@@ -9,7 +9,7 @@ transform {
 }
 JQEOT
 
-    timeout_sec = 5
```

Comment:
If this didn't cause test failures before... does it mean `jq` is missing in this test? https://github.com/snowplow/snowbridge/blob/6943238e84f4d83f36a3883c94ec4be7a7e066fb/docs/configuration_transformations_docs_test.go#L132

Reply:
Yes - it needs to be added [here](https://github.com/snowplow/snowbridge/blob/develop/docs/configuration_transformations_docs_test.go) if we want a docs test for it.